nf_core/pipeline-template/{{cookiecutter.pipeline_slug}}/docs/configuration/adding_your_own.md

```diff
@@ -77,7 +77,7 @@ singularity {
   enabled = true
 }
 process {
-  container = "docker://$wf_container"
+  container = "shub://$wf_container"
```

Comment:
`$wf_container` doesn't exist any more - should be `params.container`. Or in fact, I think we are moving in the direction of just hardcoding this as it shouldn't change much.